openstack/clustering/v1/profiletypes/results.go

```diff
@@ -0,0 +1,50 @@
+package profiletypes
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// commonResult is the response of a base result.
+type commonResult struct {
+	gophercloud.Result
+}
+
+// GetResult is the response of a Get operations.
+type GetResult struct {
+	commonResult
+}
+
+type ProfileType struct {
+	Name          string                 `json:"name"`
+	Schema        map[string]interface{} `json:"schema"`
+	SupportStatus map[string]interface{} `json:"support_status"`
```

Comment:
Is it possible to create a more strict type for both `Schema` and `SupportStatus`?

Could the _field_ `Schema` be of type `map[string]Schema` and the _type_ `Schema` be `map[string]interface{}`?

For `SupportStatus`, could the field be `map[string][]SupportStatus` and type `SupportStatus` be `map[string]interface{}` or `map[string]string`?

I'm not implying that we should create _really_ strict types that define fields such as `description`, `since`, etc (unless you feel that's manageable and possible), but something a little more tailored for the response.

Reply:
Good call on changing Schema and SupportStatus as it will make the code much cleaner. 
Don't think we should create really strict field definitions as I think will be hard to manage.